www/js/views/login.js

```diff
@@ -40,8 +40,7 @@
                                 that.model.set('name', data.name);
                                 that.model.save();
                                 FMS.isLoggedIn = 1;
-                                that.$('#password_row').hide();
-                                that.$('#success_row').show();
+                                that.rerender();
```

Comment:
I think this means that the `success_row` in login.html is never used now? As going to the router presumably means the page is redone and the "signed in" section is shown instead.